Jenkinsfile

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env groovy
+@Library('github.com/stakater/stakater-pipeline-library@v2.16.15') _
+
+releaseApplication {
+    appName = "customer"
+    appType = "maven"
+    builderImage = "stakater/builder-maven:3.5.4-jdk1.8-apline8-v0.0.3"
+    goal = "clean package"
+    notifySlack = true
+    runIntegrationTest = false
+    gitUser = "stakater-user"
+    gitEmail = "stakater@gmail.com"
+    usePersonalAccessToken = true
+    tokenCredentialID = 'GithubToken'
+    serviceAccount = "jenkins"
+    dockerRepositoryURL = 'docker-delivery.cp-stakater.com:443'
```

Comment:
please replace this with `docker.delivery.stakater.com:443`

Reply:
Done.